docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_9.md

```diff
@@ -0,0 +1,126 @@
+# ExperimentalDetectronGenerateProposalsSingleImage {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_9}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-9*
```

Comment:
Also, I propose to think of removing `Experimental` prefix in the name because now we evidently have several frameworks with such operation not only Caffe2.